src/client/service-components/EditableInput.js

```diff
@@ -0,0 +1,67 @@
+import React, {useState, useContext} from 'react';
+import {UserSettingsContext} from '../context-store/user-settings-context.js';
+import './EditableInput.css';
+
+export const EditableInput = (props) => {
+    const {theme} = useContext(UserSettingsContext);
+    const [isEdit, setIsEdit] = useState(false);
+    const [isInputEmpty, setIsInputEmpty] = useState(false);
+    const [text, setText] = useState(props.text);
+    const [previousText, setPreviousText] = useState(props.text);
+
+    const handleEditClick = () => {
+        setIsEdit(true);
+    };
+
+    const updateColumnName = ({target}) => {
+        setText(target.value);
+        target.value ? setIsInputEmpty(false) : setIsInputEmpty(true);
```

Comment:
nit: it can be even easier 😄

Reply:
Done. Commit a63b460e6e3992d3d69e3c382075bb57d9073c39